### DISCUSSION TITLE
My ticket has no response at all, even I have opened the ticket from 3 month ago.

### DISCUSSION THREAD

**Discussion by haikaru17** [community/community#137272] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Does anyone else have the same experience as me with no response at all?

**Comment by ebndev** [comment]
Hi @haikaru17,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
